templates/channels/channel_configuration.haml

```diff
@@ -510,6 +510,47 @@
       %code= ip_address
 
 
+  -elif object.channel_type == 'FCM'
+    %h4
+      -trans "To use your Firebase Cloud Messaging channel you'll have to POST to the following URLs with the parameters below."
+
+    .clearfix
+      .config-value
+        .name
+          -trans "Name:"
+        .value {{ object.name }}
+      .config-value
+        .name
+          -trans "Address/Key:"
+        .value {{ object.address }}
+      .config-value
+        .name
+          -trans "Notification title:"
+        .value {{ object.config_json.FCM_TITLE }}
+
```

Comment:
Yes, it's need set the key in the app. But, it's already displayed as channel address, what do you think?